utils/contentManager.js

```diff
@@ -0,0 +1,8 @@
+const uppercaseFirst = word => {
```

Comment:
Created this so that when lower-case value from Database is taken out, it's made into uppercase on Hero Card